### ISSUE TITLE
production 'A : A @ B' not handled correctly

### ISSUE THREAD

**Issue by jmdyck** [rbuckton/grammarkdown#12]
[Section 12.11.3 of the ES6 spec](http://www.ecma-international.org/ecma-262/6.0/#sec-binary-bitwise-operators-runtime-semantics-evaluation) begins (in ecmarkup):
```
<p>The production <emu-grammar>A : A @ B</emu-grammar>, where @ is ...
```
grammarkdown does not handle this production correctly, translating it to:
```
    A:A

    B:[empty]
```

**Comment by rbuckton** [comment]
@bterlson Is there a specific emu tag I should associate with the `@` placeholder in the production above, or just emit it verbatim/treat it as a terminal.

**Comment by bterlson** [comment]
I think it's a non-terminal, actually - just one that isn't formally defined as part of the grammar and used as a convenience when referring to all the terminals it expands to...

**Comment by rbuckton** [comment]
So we even need a change here? We should instead write it as:

```gmd
A : A `@` B
```

**Comment by bterlson** [comment]
I think `@` is conceptually a non-terminal that expands to the various binary op terminals, so I think backticks are inappropriate?

**Comment by rbuckton** [comment]
So assuming I just add @ as a "placeholder" symbol, would I still emit @ as `<emu-t>@</emu-t>`?

**Comment by bterlson** [comment]
`<emu-nt>@</emu-nt>` I think, or maybe `<emu-nt placeholder>@</emu-nt>` if we want to consider @ a special form. I would be happy with either.

**Comment by rbuckton** [comment]
Fixed in c299b3d5.
